ikarus/finiteelements/feresulttypes.hh

```diff
@@ -203,7 +203,7 @@ public:
     } else
       return value_;
   }
-  explicit ResultWrapper() = default;
+  explicit ResultWrapper() { value_.setZero(); }
```

Comment:
Can you make this again default and change it in line 219? There, you can init with `StoredType::Zero()`.